ETC.tsv

```diff
@@ -97,9 +97,9 @@ xETC_20150317_000096	The brightly colored eyes make these monsters hard to miss.
 ETC_20150317_000097	Yellow Griba
 ETC_20150317_000098	Large Red Griba
 ETC_20150317_000099	Big Griba
-xETC_20150317_000100	While there are valuations for this creature, this monster is actually hard to eat.
+xETC_20150317_000100	While there is demand for this creature's meat, this monster is actually hard to eat.
 ETC_20150317_000101	Big Blue Griba
-xETC_20150317_000102	A large loathsome creature that is intentionally hunted.
+xETC_20150317_000102	A large loathsome creature that is unremittingly hunted.
```

Comment:
I agree that "Unremittingle" is a bit overly complicated, but if something is hunted it's pretty much a given that it's intentional so something should change.